glue/clients/util.py

```diff
@@ -4,11 +4,15 @@
 from functools import partial, wraps
 
 import numpy as np
+import pandas as pd
+from matplotlib.dates import date2num, AutoDateLocator, AutoDateFormatter
+import datetime
 from matplotlib.ticker import AutoLocator, MaxNLocator, LogLocator
 from matplotlib.ticker import (LogFormatterMathtext, ScalarFormatter,
                                FuncFormatter)
 from matplotlib.backends.backend_agg import FigureCanvasAgg
 from ..core.data import CategoricalComponent
+from colorsys import hls_to_rgb
```

Comment:
nit: imports from the python standard lib go in the first block of imports (with functools above)